### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MaxSchmide**
- [DEMO LINK](https://maxschmide.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://maxschmide.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,24 @@
...
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <a class="nav__item is-active">Apple</a>
```
Comment (ManoylovAC) [inline src/index.html:20]: oops, you missing `li` tags, please check it out again

**Review by ManoylovAC — CHANGES_REQUESTED** [review]
Hi 👋 
Good time to some improvements.
Please recheck requirements, for example your layout haven't fit on different widths (1024px, 1200px). And also fix my suggestion in comment I left
Hint: for adaptive styles you can youse media-queries

**Comment by MaxSchmide** [comment]
> Hi 👋 Good time to some improvements. Please recheck requirements, for example your layout haven't fit on different widths (1024px, 1200px). And also fix my suggestion in comment I left Hint: for adaptive styles you can youse media-queries

What exactly is wrong on different screen sizes? I looked at mockup and all difference is distance between logo and nav, which auto fixing by flex and justify-content: space-between. Other spaces are the same. So, what I missed?

File: src/style.css [hunk 2]
```diff
@@ -1 +1,74 @@
-/* Styles go here */
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
```
Comment (tetiana-ivanchenko) [inline src/style.css:7]: image

File: src/style.css [hunk 3]
```diff
@@ -1 +1,74 @@
...
+
+html {
+  font-family: "Roboto", sans-serif;
```
Comment (tetiana-ivanchenko) [inline src/style.css:10]: image

File: src/style.css [hunk 4]
```diff
@@ -1 +1,74 @@
...
+  padding: 23px 0;
+  cursor: pointer;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (tetiana-ivanchenko) [inline src/style.css:58]: move repeatable colors in the variables

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -5,10 +5,44 @@
...
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="MOYO logo"
+        class="logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active">Apple</a>
```
Comment (anstsot) [inline src/index.html:21]: all links should have `href` attribute. please add it to all needed elements

**Review by anstsot — CHANGES_REQUESTED** [review]
Let’s add the last fixes to improve your code

File: src/index.html [hunk 6]
```diff
@@ -5,10 +5,50 @@
...
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            class="nav__link"
+            href="#"
+            data-qa="hover"
+            >
+            laptops & computers
+            </a>
```
Comment (tetiana-ivanchenko) [inline src/index.html:36]:
```suggestion
            <a
              class="nav__link"
              href="#"
              data-qa="hover"
            >
              laptops & computers
            </a>
```

**Pull request closed without merging**